src/Disks/S3/registerDiskS3.cpp

```diff
@@ -118,6 +118,9 @@ void registerDiskS3(DiskFactory & factory)
 
         cfg.connectTimeoutMs = config.getUInt(config_prefix + ".connect_timeout_ms", 10000);
         cfg.httpRequestTimeoutMs = config.getUInt(config_prefix + ".request_timeout_ms", 5000);
+        cfg.maxConnections = config.getUInt(config_prefix + ".max_connections", 25);
+        cfg.followRedirects = config.getBool(config_prefix + ".follow_redirects", false);
```

Comment:
Why we set it to false? The default value is true https://github.com/ClickHouse-Extras/aws-sdk-cpp/blob/master/aws-cpp-sdk-core/source/client/ClientConfiguration.cpp#L66